### PR TITLE
Add back auto-refresh on dashboard (addresses #2668)

### LINKED ISSUE #2668: Requesting documentation update for Dashboard Autorefresh
**PR:**  The feature is in tools!

_Describe the new feature here._

Include pertinent details, such as:

- What the feature does and why it is useful
It allows users to auto-refresh dashboards based on their specifications ( duration, interval .. etc)

- How to use the feature (via CLI, UI, API)
On the UI through the enable auto refresh button. Screenshots below

influxdb aws influxdata io_orgs_844910ece80be8bc_dashboards_0698872fccde4000_lower=now%28%29%20-%201h

influxdb aws influxdata io_orgs_844910ece80be8bc_dashboards_0698872fccde4000_lower=now%28%29%20-%201h (2)
influxdb aws influxdata io_orgs_844910ece80be8bc_dashboards_0698872fccde4000_lower=now%28%29%20-%201h (3)

##### Relevant URLs
- _Provide relevant URLs (issues, community threads, existing doc pages, etc.)_
[Epic Link](https://app.zenhub.com/workspaces/applicationui-5e3b05772922e316b3a210a4/issues/influxdata/ui/1207)

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #2668

### CODE REVIEW

**Comment by daviddkkim** [comment]
@noramullen1 Would you like any images or GIFs for this?

**Comment by noramullen1** [comment]
> @noramullen1 Would you like any images or GIFs for this?

@daviddkkim No thanks! we avoid images unless absolutely necessary because of the maintenance. :)

File: content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md [hunk 1]
```diff
@@ -37,11 +37,19 @@ Click the timezone dropdown to select a timezone to use for the dashboard. Selec
...
+#### Manually refresh a single dashboard cell
 
 1. Click the **{{< icon "gear" >}}** on the dashboard cell you want to refresh.
 2. Click **{{< icon "refresh" >}} Refresh**.
 
+### Automatically refresh dashboard
+
+Click **Enable Auto Refresh**. In the window that appears, configure the following options:
+  - **Until**: Select **Indefinite** to automatically refresh continuously. Select **Custom** to set a time to stop automatically refreshing.
+  - **Inactivity Timeout**: Set how long the dashboard should be inactive for to time out and stop automatically refreshing.
```
Comment (kelseiv) [inline content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md:49]:
```suggestion
  - **Inactivity Timeout**: Set how long the dashboard should be inactive before timeout stops automatically refreshing.
```

Comment (kelseiv) [reply content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md:49]: is this change accurate?

File: content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md [hunk 2]
```diff
@@ -37,11 +37,19 @@ Click the timezone dropdown to select a timezone to use for the dashboard. Selec
```
Comment (kelseiv) [inline content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md:37]: Should these changes be in both Cloud and OSS docs?

Comment (daviddkkim) [reply content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md:37]: @ShmuelLotman is auto-refresh in Both Cloud and OSS?

File: content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md (hunk 1 above)
Comment (ShmuelLotman) [reply content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md:49]: Maybe its worth saying "user should be inactive" or "page should be inactive" to be clearer about what is the trigger for the timeout?

**Review by ShmuelLotman — APPROVED** [review]
LGTM. Just one minor thought, but non blocking

File: content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md (hunk 1 above)
Comment (daviddkkim) [reply content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md:49]: How about `"Set how long the dashboard should be inactive before the auto-refresh times out and stops automatically refreshing" ` ?

Comment (ShmuelLotman) [reply content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md:49]: "the dashboard should be inactive" is the part im not sure about whether the word should be dashboard or user

Comment (daviddkkim) [reply content/influxdb/v2.0/visualize-data/dashboards/control-dashboard.md:49]: yeah sorry I didn't see your comment because I didn't refresh github

**Review by kelseiv — APPROVED** [review]

**Review by daviddkkim — APPROVED** [review]
lgtm

**Comment by noramullen1** [comment]
@daviddkkim, do we have a timeline for this? Thanks.

**Comment by daviddkkim** [comment]
> @daviddkkim, do we have a timeline for this? Thanks.

@ShmuelLotman do you know when autorefresh will propagate to everywhere in prod?

**Pull request merged**